app/components/activity/ActivityList.js

```diff
@@ -90,27 +90,27 @@ class ActivityList extends React.Component {
 		const { currentActivities } = this.state;
 
 		const filteredActivities = currentActivities.filter(item => {
-      			return item.title.toLowerCase().trim().indexOf(term.toLowerCase().trim()) > -1;
+						return item.title.toLowerCase().trim().indexOf(term.toLowerCase().trim()) > -1;
 		});
-		
+
 		this.setState({ filteredActivities })
 	}
 
 	renderListItem = ({ item, index }) => {
 		const addIcon = (
 			<Icon
 				style={ ActivityStyles.activityListItemIcon }
-				name="add-circle-outline"
-				color="black"
+				name="ios-add-circle"
+				color="#0288D1"
```

Comment:
Should consider storing these as constants in the `styles/Colours.js` file so that we can re-use colours or change them without going through all files